### PR TITLE
Dynamic search on advisor and advisor/client #177086150

### PR DESCRIPTION
**Pull request by ericanderson91**
@jon-oblander adding you as a review since you worked this in bxhealth, I did not make the API call a post because it seems the filter parameter was already used and we only filter on uids in BXF so adding that to the url seemed easier.

### CODE REVIEW

File: bxfinance-scenario1/src/pages/advisor/index.js [hunk 1]
```diff
@@ -68,11 +71,21 @@ const SearchAutocomplete = () => {
...
+      pingData.getSearchableUsers({ limit: '20', searchTerm: value})
+        .then(response => response.json())
+        .then(jsonSearchResults => {
+          const filteredSuggestions = jsonSearch.findValues(jsonSearchResults._embedded.entries, 'uid');
+
+          setFilteredSuggestions(filteredSuggestions);
+          setDisplaySuggestions(true);
+        });
+    } else {
+      console.log('else');
```
Comment (jon-oblander) [inline bxfinance-scenario1/src/pages/advisor/index.js:85]: console.log still needed?

File: bxfinance-scenario1/src/components/Integration/PingData.js [hunk 2]
```diff
@@ -90,7 +90,7 @@ class PingData {
     @param {number} limit The max number of records to return. You could get less based on number of records found or PingDirectory-configured limits. Defaults to 100.
     @return {object} The response JSON object.
     */
-    getSearchableUsers({ searchScope = "singleLevel", limit = "100" }) {
+    getSearchableUsers({ searchScope = "singleLevel", limit = "100", searchTerm = '' }) {
```
Comment (jon-oblander) [inline bxfinance-scenario1/src/components/Integration/PingData.js:93]: trivial but can we make it consistent with single vs double quotes

File: bxfinance-scenario1/src/pages/advisor/index.js (hunk 1 above)
Comment (ericanderson91) [reply bxfinance-scenario1/src/pages/advisor/index.js]: Nope, good call

**Review by jon-oblander — APPROVED** [review]
Reviewed.

**Pull request merged**